src/constants/index.js

```diff
@@ -1,5 +1,2 @@
-export const LoadingStates = {
-  idle: "idle",
-  loading: "loading",
-  loaded: "loaded"
-};
+export * from "./hypothesis-statuses";
+export * from "./incident-priority";
```

Comment:
Blanket imports and exports are usually not a good idea. Clashes. Difficulty in reading code (where is the symbol imported from) etc.